applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp

```diff
@@ -2492,7 +2270,7 @@ namespace Kratos
 				iValue(0, 1) = iValue(1, 0) = data.generalizedStresses[2] -
 					data.generalizedStresses[5];
 				iValue(0, 2) = iValue(2, 0) = 0.0;
```

Comment:
uff, i don't understand this statement (i wouldn't be able to tell the order in which it is executed...)